### ISSUE TITLE
Can airflow be triggered via webhooks?

### ISSUE THREAD

**Issue by dkinzer** [tulibraries/grittyOps#79]
I would like to look into possibilities of using webhooks to trigger an airflow dag.

* Is it even possible to trigger dags with webhooks?
* Who has access to creating alma webhooks (is this something devs can get access to)?

Ref:
https://developers.exlibrisgroup.com/alma/integrations/webhooks/

https://developers.exlibrisgroup.com/alma/integrations/webhooks/events/

https://developers.exlibrisgroup.com/alma/integrations/webhooks/anatomy/

**Comment by relaxing** [comment]
Yes we can do that, assuming an alma webhook exists for the event we're interested in.
